src/core/file_sys/fs.cpp

```diff
@@ -0,0 +1,76 @@
+#include "fs.h"
+
+#include <algorithm>
+
+namespace Core::FileSys {
+
+constexpr int RESERVED_HANDLES = 3; //first 3 handles are stdin,stdout,stderr
```

Comment:
```suggestion
constexpr int RESERVED_HANDLES = 3; // First 3 handles are stdin,stdout,stderr
```
Comments should start with capital letter and have space for better readability

Reply:
ok